lib/checks/label/duplicate-img-label.js

```diff
@@ -1,4 +1,4 @@
-const text = axe.commons.text.visible(virtualNode, true).toLowerCase();
+const text = axe.commons.text.visibleVirtual(virtualNode, true).toLowerCase();
```

Comment:
That's not how I understood it. This made more sense to me, as this way, you don't have to look up the virtual node again, which is what you'd need to do if you just call `.visible`.